src/components/shared/components/Layout/components/NavigationLayout.tsx

```
@@ -1,170 +1,127 @@
+                            key={item.label}
+                            item={item}
+                            isOpen={isLeftDrawerOpen}
+                            onClick={closeMobileDrawer}
+                          />
+
+                          {!!navSection && "children" in navSection && (
+                            <List className="w-full">
+                              {navSection.children
+                                .filter((page) => page.route !== item.url)
+                                .map((page) => (
+                                  <div className="px-3 pl-12 sm:pr-6 pb-3 sm:pb-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)